aws-typescript/tests/infra.spec.ts

```diff
@@ -0,0 +1,51 @@
+import * as pulumi from "@pulumi/pulumi";
+import "jest";
+
+// Test helper to convert a Pulumi Output to a Promise.
+// This should only be used in tests.
```

Comment:
I think we could elaborate a bit more here why this should only be used in tests, maybe something along the lines of:

```suggestion
// This should only be used in tests. Outputs track more information
// than Promises, such as the dependencies between resources.
// https://www.pulumi.com/docs/iac/concepts/inputs-outputs/#outputs
```